setup_dev_env.sh

```diff
@@ -117,6 +117,10 @@ if [ $ERR -ne 0 ]; then
     exit $ERR
 fi
 
+DOCKER_GID=$(id -g $USER)
+if [ -z "$DOCKER_GID" ]; then
+    DOCKER_GID=$(id -u $USER)
```

Comment:
Under what circumstances would id -g $USER return nothing? Is setting the GID to the UID the right thing to do?

AFAIK, they are different ID spaces.

Reply:
It should never fail. This was defensive programming. And yes, they are different spaces, which was the original problem. In the case of this failure, I was just reverting to the prior (incorrect) behavior, out of lack of knowing what else to do. Perhaps there is a more elegant solution, which would be to set the entire USER:GID string here instead of just setting the GID.

Reply:
Understood :) 

I like your new approach of omitting the GID if not present.